lfucg-exactions/client/js/components/AccountExisting.js

```diff
@@ -27,9 +26,26 @@ class AccountExisting extends React.Component {
         const {
             currentUser,
             accounts,
-            onAccountQuery,
+            plats,
```

Comment:
@JMStewart00 Instead of doing the `plats &&` kind of checks below, you can say `plats = []` here. This way you know you are always working with an array.